test/functional/feature_bip68_sequence.py

```diff
@@ -26,13 +26,14 @@
 # RPC error for non-BIP68 final transactions
 NOT_FINAL_ERROR = "non-BIP68-final (code 64)"
 
+DUMMY_P2WPKH_SCRIPT = CScript([b'a' * 21])
```

Comment:
Could move this to a common place like `./test/functional/test_framework/util.py` and import it from there?

Reply:
The misery is that I can't find an appropriate place where to put the constant into. Importing the module `script` in `util.py` leads to a circular dependency (`script` imports `messages`, which in turn imports `util`), while declaring the constant in `script.py` doesn't work as on top `CScript` is not defined yet. Any other module within `test_framework` doesn't seem to fit thematically. Any suggestion?

Reply:
You can create a new file?